amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```
@@ -2092,6 +2212,289 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  ## Holds
+  '/holds/{holdId}':
+    get:
+      tags:
+        - holds
+      summary: Get a hold
+      description: |
+        Gets information for hold with id **holdId**.
+      operationId: getHold
+      parameters:
+        - $ref: '#/parameters/holdIdParam'
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/HoldModelEntry'
+        '400':
+          description: |
+            Invalid parameter: **holdId** is not a valid format
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to read **holdId**
+        '404':
+          description: "**holdId** does not exist"
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    put:
+      tags:
+        - holds
+      summary: Update a hold
+      description: |
+        Updates the hold with id **holdId**. For example, you can rename a hold:
+        ```JSON
+        {
+          "name":"My new name",
+          "description":"My new description",
+          "reason":"My new reason"
```

Perhaps this would match the description better like:
```suggestion
          "description":"Existing description",
          "reason":"Existing reason"
```